admin-dev/themes/default/js/bundle/module/module.js

```diff
@@ -369,10 +365,10 @@ var AdminModuleController = function () {
                  $(_that.moduleImportProcessingSelector).fadeIn();
              });
           },
-          processing: function (file, response) {
```

Comment:
If you can re-add this deletion as comment just before declaration it would be great, I left these on purpose to easily know callback's arguments if we need them in the future, without going through code and/or documentation